src/helpers/mathjax.ts

```diff
@@ -95,9 +95,26 @@ function typesetDocument(root: Element, windowImpl: Window) {
   );
 }
 
+let recursiveCounter = 0;
```

Comment:
this isn't gonna work if its global, you should pass it as `remainingTries` to the function, default it to 5 and subtract one each time you recurse